.gitignore

```diff
@@ -46,7 +46,8 @@ coverage.xml
 *.cover
 .hypothesis/
 .pytest_cache/
-*.json
```

Comment:
Why do we have this?

Reply:
Why wouldn't we have this? Note that `.gitignore` is only for when you do `git add *` or some other wildcard search. This forces people to type `git add filename` for an explicit change to a filename intended to be pushed.

Reply:
I was wondering if you were tracking some specific files with this? Do json files appear on in some part of our processing (build or something) or is it just for the gin config files? 
That was the sense of the question, not that I think is a bad idea. I should have been more explicit.